client/src/pages/Home.js

```diff
@@ -1,15 +1,14 @@
 // @flow
```

Comment:
The `: Node` syntax in this file was actually meant for Flow, not TS. Since we'll be using vanilla JS for this project, we can remove this line for all files.